draft-ietf-ice-rfc5245bis.xml

```diff
@@ -2792,29 +2793,34 @@ repeats the connectivity check that produced this valid pair (by enqueueing
 the pair that generated the check into the TRIGGERED CHECK QUEUE), this time with
 the USE-CANDIDATE attribute. The connectivity check should succeed (since the
 previous did), causing the nominated flag value of that and only that pair
-to be set to true.
-</t>
-<t>Eventually, there will be only a single nominated pair in the VALID LIST for
-each component. Once the state of the CHECK LIST is set to Completed, that exact
-pair is selected by ICE for sending and receiving media for that component.
+to be set to true. However, if the connectivity check fails <xref target="sec-client-failure"/>, 
+the controlling agent MUST remove the candidate pair from the VALID LIST, set 
+the candidate pair state to Failed and set the CHECK LIST state to Failed.</t>
+<t>Eventually, if the nominations succeed, there will be only a single nominated 
+pair in the VALID LIST for each component. Once the state of the CHECK LIST is 
+set to Completed, that exact pair is selected by ICE for sending and receiving 
```

Comment:
Instead of "that exact pair is selected", I think "the controlled agent MUST select the nominated pair" would be more clear.